cmd/modelcmd/modelcommand.go

```diff
@@ -94,15 +96,15 @@ type ModelCommandBase struct {
 	// about controllers, models, etc.
 	store jujuclient.ClientStore
 
-	// _modelName, _modelType, _modelGeneration and _controllerName hold the
-	// current model and controller names, model type and generation. They
-	// are only valid after maybeInitModel is called, and should in general
-	// not be accessed directly, but through ModelName and ControllerName
-	// respectively.
-	_modelName      string
-	_modelType      model.ModelType
-	_activeBranch   string
-	_controllerName string
+	// _modelIdentifier, _modelType, _modelGeneration and _controllerName hold
+	// the current model identifier, controller name, model type and branch.
+	// They are only valid after maybeInitModel is called, and should in
+	// general not be accessed directly, but through ModelIdentifier and
+	// ControllerName respectively.
```

Comment:
If we are doing model name, is it worth doing controller-by-uuid as well? Probably not in this patch, but it does seem like a reasonable consistency thing.
That said, we log model-uuids (cause we have them easily accessible), but we don't log controller-uuids particularly often. So there is less impetus for it.